src/example-types/groupedMetric.js

```diff
@@ -42,15 +38,15 @@ module.exports = {
                     : {
                         field: agg.field,
                       },
-                  agg.data
+                  agg
```

Comment:
I don't think this is right because it is inside `agg`, which is in the aggs array.